tests/test_regression.py

```diff
@@ -0,0 +1,91 @@
+import numpy as np
+
+# import pandas as pd
+import pytest
+from typing import Union, Sequence
+
+from cleanlab.regression import rank
+
+ArrayLike = Union[np.ndarray, Sequence]
```

Comment:
how is this being used in this file?

Reply:
This is not used. I have deleted this in [feb5797](https://github.com/cleanlab/cleanlab/pull/572/commits/feb5797a66e7c5fe6128c63ff20d2ecf98474333)